src/ValidationResult.res

```diff
@@ -0,0 +1,5 @@
+@unboxed type rec t = Any('value): t
```

Comment:
I don't seem to understand this line. Can you explain it in more details? And why should we wrap the handler with Any?